view/src/planner.rs

```diff
@@ -326,6 +327,32 @@ impl<R: RngCore + CryptoRng> Planner<R> {
         self
     }
 
+    /// Perform an ICS-20 withdrawal
+    #[instrument(skip(self))]
+    pub fn ics20_withdrawal(
+        &mut self,
+        destination_chain_id: String,
+        destination_chain_address: String,
+        denom: Denom,
+        amount: Amount,
+        timeout_height: u64,
+        timeout_timestamp: u64,
+        return_address: Address,
+        source_channel: ChannelId,
+    ) -> &mut Self {
+        self.action(ActionPlan::WithdrawalPlan(Ics20WithdrawalPlan {
+            destination_chain_id,
+            destination_chain_address,
+            denom,
+            amount,
+            timeout_height,
+            timeout_timestamp,
+            source_channel,
+            return_address,
+        }));
+        self
+    }
```

Comment:
Rather than duplicating all of the fields, I think this method should just take the ICS20 withdrawal action itself, letting the caller construct it themselves with named fields if they want, or pass an object if they already had one.